src/redux/books/books.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Book from './book';
+
+function Books() {
+  return (
+    <div>
+      <Book />
+      <form className="addForm">
+        <label htmlFor="title">
+          <input id="title" placeholder="Title" Required />
+        </label>
+        <label htmlFor="author">
+          <input Id="author" placeholder="Author" Required />
+        </label>
+        <button type="submit" className="add btn">Add</button>
+      </form>
```

Comment:
- [ ] Please move this form to it's own standalone component, this will make it easier implementing adding a book in the next milestone.

Reply:
Thank you for your helpful review! I have made this a standalone component.